ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -168,7 +169,9 @@ object ParallelIndexerFactory {
             )
             .use { dbDispatcher =>
               dbDispatcher
-                .executeSql(metrics.daml.parallelIndexer.initialization)(storageBackend.initialize)
+                .executeSql(metrics.daml.parallelIndexer.initialization)(
```

Comment:
I am missing the part what we discussed before, and moves append only ledger initialization here. 
Will this be part of some other task or the same? Will this be part of a different PR?

Reply:
I wasn't planning on doing this as part of DPP-460

Reply:
sure no probs, created DPP-551 for this, I will do it as part of HA stabilization